include/aws/crt/Types.h

```diff
@@ -77,9 +77,7 @@ namespace Aws
          * A conversion function should be provided to do the type conversion
          */
         template <typename RawType, typename TargetType>
-        AWS_CRT_CPP_API Vector<TargetType> ArrayListToVector(
-            const aws_array_list *array,
-            TypeConvertor<RawType, TargetType> conv)
+        Vector<TargetType> ArrayListToVector(const aws_array_list *array, TypeConvertor<RawType, TargetType> conv)
```

Comment:
This removal makes much sense. It's implemented in header file, no need to export.